mobile/screens/AddTrainingLog.tsx

```diff
@@ -1,18 +1,89 @@
 import { StatusBar } from "expo-status-bar";
-import React, { useEffect } from "react";
-import { BackHandler, StyleSheet, Text, View } from "react-native";
+import React, { useEffect, useState } from "react";
+import {
+  BackHandler,
+  FlatList,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+import HeaderDate from "../components/HeaderDate";
 
 export default function AddTrainingLogScreen(props: any) {
-  useEffect(() => {
-    BackHandler.addEventListener("hardwareBackPress", function () {
-      props.navigation.navigate("User Dashboard");
-      return true;
-    });
-  }, []);
+  const [totalHours, setTotalHours] = useState("");
+  const [skills, setSkills] = useState([
+    { id: 1, skill: "post", selected: "false" },
+    { id: 2, skill: "heel", selected: "false" },
+    { id: 3, skill: "jump", selected: "false" },
+  ]);
+  const [error, setError] = useState("");
+  const [skillsPlayed, setSkillPlayed] = useState("");
 
   return (
     <View style={styles.container}>
-      <Text>Add Training Log Screen</Text>
+      <View>
+        <HeaderDate />
+        <View>
+          <View>
+            <Text style={styles.label}>Total hrs*</Text>
+            <TextInput style={styles.input} onChangeText={setTotalHours} />
+          </View>
+          <View>
+            <Text style={styles.label}>Note</Text>
+            <TextInput style={styles.input} />
+          </View>
+          <View>
+            <Text style={styles.label}>Skill Played*</Text>
+            <FlatList
+              horizontal
+              data={skills}
+              renderItem={({ item }) => (
+                <TouchableOpacity
+                  style={styles.multiSelectButton}
+                  onPress={() => {
+                    setSkillPlayed(item.skill);
+                  }}
+                >
+                  <Text>{item.skill}</Text>
+                </TouchableOpacity>
```

Comment:
Would you be able to indicate somehow that a skill is selected -- I know the designers didn't include this but maybe make the border of the selected skill darker or soemthing (the UI doesn't make it super clear that a skill has been selected)